src/arload.py

```diff
@@ -123,6 +125,8 @@ def optarg(argflag, argv, pypname):
         if o in ('-h', '--help'): msgs.usage(None)
         elif o in ('-c', '--cpus'): argflag['run']['ncpus'] = a
         elif o in ('-v', '--verbose'): argflag['out']['verbose'] = int(a)
+    for a in arg:
```

Comment:
I think this should be in the above for loop (i.e. the for loop that starts on line 124). Note that o refers to option and a refers to the argument of that option:
elif o in ('m', 'use_masters'): argflag['masters']['use'] = True